src/util/std_code.h

```diff
@@ -48,6 +48,26 @@ class codet:public exprt
     set_statement(statement);
   }
 
+  /// \param statement: Specifies the type of the `codet` to be constructed,
+  ///   e.g. `ID_block` for a \ref code_blockt or `ID_assign` for a
+  ///   \ref code_assignt.
+  explicit codet(const irep_idt &statement, operandst &&_op)
+    : exprt(ID_code, empty_typet())
+  {
+    set_statement(statement);
+    operands() = std::move(_op);
+  }
+
+  /// \param statement: Specifies the type of the `codet` to be constructed,
+  ///   e.g. `ID_block` for a \ref code_blockt or `ID_assign` for a
+  ///   \ref code_assignt.
+  explicit codet(const irep_idt &statement, const operandst &_op)
```

Comment:
Having done a bunch of profiling and studied the profiles: we can likely get away with a single constructor taking `operandst` by value, and then `std::move` it in place.

Reply:
I suspected as much, and like the reduction in noise.